test/search-query.ts

```diff
@@ -4,34 +4,34 @@ import './fixtures/globals';
 import SearchQuery from '../source/github-helpers/search-query';
 
 test('.get', t => {
-	const query = new SearchQuery(new URLSearchParams('?q=wow'));
+	const query = new SearchQuery('?q=wow');
 	t.is(query.get(), 'wow');
 });
 
 test('.getQueryParts', t => {
-	const query = new SearchQuery(new URLSearchParams('?q=cool+is%3Aissue'));
+	const query = new SearchQuery('?q=cool+is%3Aissue');
```

Comment:
I'd rather see a more readable
```suggestion
	const query = new SearchQuery(['q', 'cool is:issue']);
```

Reply:
Sure, so just accept all `URLSearchParams` options?
```js
URLSearchParams(init?: string | string[][] | Record<string, string> | URLSearchParams)
```

*Would be this then*
```js
new SearchQuery([['q', 'cool is:issue']]);
```

But it would also work to just pass the not escaped string if it is just for readability in the tests
```js
new SearchQuery('q=cool is:issue');
```

Reply:
I just changed `SearchQuery` to accept all `URLSearchParams` options.
And then used the `{q: 'query'}` parameter *(thought it was more readable than the entries option)*.

Hope that is okay otherwise I can also change it back or to something else.